backend/grants/admin.py

```diff
@@ -240,6 +243,79 @@ def send_reply_email_waiting_list_update(modeladmin, request, queryset):
         messages.info(request, f"Sent Waiting List update reply email to {grant.name}")
 
 
+@admin.action(description="Send voucher via email")
+def send_voucher_via_email(modeladmin, request, queryset):
+    is_filtered_by_conference = (
+        queryset.values_list("conference_id").distinct().count() == 1
+    )
+
+    if not is_filtered_by_conference:
+        messages.error(request, "Please select only one conference")
+        return
+
+    count = 0
+    for grant in queryset.filter(pretix_voucher_id__isnull=False):
+        send_grant_voucher_email(grant)
+        count = count + 1
+
+    messages.success(request, f"{count} Voucher emails scheduled!")
+
+
+def _generate_voucher_code() -> str:
+    charset = list("ABCDEFGHKLMNPQRSTUVWXYZ23456789")
+    random_string = get_random_string(length=20, allowed_chars=charset)
+    return f"GRANT-{random_string}"
```

Comment:
I am sure we have a similar function for speakers, we should refactor and share it in the two places?

```suggestion
def _generate_voucher_code(prefix: str) -> str:
    charset = list("ABCDEFGHKLMNPQRSTUVWXYZ23456789")
    random_string = get_random_string(length=20, allowed_chars=charset)
    return f"{prefix}-{random_string}"
```